mlir/test/Conversion/ConvertToAIR/condense_memref_ops_to_air_memcpy.mlir

```diff
@@ -0,0 +1,92 @@
+//===- condense_memref_ops_to_air_memcpy.mlir ------------------*- MLIR -*-===//
+//
+// Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+// SPDX-License-Identifier: MIT
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Thanks for the comment. 

`-canonicalize` in this test removes duplicates of `arith.constant` and could make the tests a lot shorter. Checking the entries of wraps and strides is needed in this test, and those are SSA values returned from `arith.constant` ops. If without `-canonicalize` then the FileCheck checks shall go over 100 lines...

Reply:
Ok, I understand. I wonder, if you ran constant subexpression elimination then you wouldn't need to check every line because the arith.constants would all be unique.

Reply:
No because the air.launch air.segment and air.herd ops have `isolateFromAbove` interface, so the constant ops are not really unique across those ops...